NewForce-at-Mountwest/bangazon-site-take-2-centaurs
# Db ticket 3

When a user creates a new product, within the form there is a checkbox for local delivery. If that checkbox is checked then the boolean for that column in the database is switched from false to true and that data will be stored for later.

Future iterations will make it to where the user must type in a city once that checkbox is checked. Once that is done, a search for cities in the search field will display only products that have local delivery available.

RELATED TICKETS

#3 Seller can specify local delivery available

STEPS TO TEST

git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout db-ticket-3 to access my branch.
If you don't see the most recent version of their files, try git pull origin db-ticket-3 on your server on my branch and make sure everything looks okay.

Create a new product, check the Local Delivery Available checkbox.

Gaze upon and be awed by partial working functionality!

Checklist:
[] I have commented my code, particularly in hard-to-understand areas
[] I have made corresponding changes to the documentation
[]My changes generate no new errors
[] I have added tests that prove my fix is effective or that my feature works
[] New and existing unit tests pass locally with my changes
